schedule/views.py

```diff
@@ -315,14 +315,22 @@ def api_occurrences(request):
         convert = lambda d: datetime.datetime.utcfromtimestamp(float(d))
     start = utc.localize(convert(request.GET.get('start')))
     end = utc.localize(convert(request.GET.get('end')))
-    calendar = get_object_or_404(Calendar, slug=request.GET.get('calendar_slug'))
+    calendar_slug = request.GET.get('calendar_slug')
+    if calendar_slug:
```

Comment:
Shouldn't this be:
```
if calendar_slug:
    calendars = [get_object_or_404(Calendar, slug=calendar_slug)]
```
Since you've already confirmed at this point that you have the calendar slug. No need to retrieve it from the request object again.